config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  root "users#index"
+  resources :posts
+  resources :users
```

Comment:
- [ ] Based on the [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project_v1_1.md#make-sure-that-when-you-run-the-rails-routes-command-in-your-console-you-can-see-the-following-output), Kindly add in your routes the `show` routes so users can access to the show page just like the index page👍🏿